test/it/api_auth_it.js

```diff
@@ -588,6 +590,35 @@ describe('Application.authenticateApiRequest',function(){
       assert.equal(scopeFactoryArgs[1][1],requestedScopes[1]);
     });
 
+    it('should call the scope factory with the requested scope when scopeFactory is given a callback', function(done) {
+      var requestObject = {
+        headers: {
+          'authorization': 'Basic ' + new Buffer([apiKey.id,apiKey.secret].join(':')).toString('base64')
+        },
+        method: 'POST',
+        url: '/some/resource?grant_type=client_credentials&scope='+requestedScope
```

Comment:
'constancy'? ;) But yeah, I copy-pasted that block, so that's why. But I'll fix that.